src/common/dsp/modulators/FormulaModulationHelper.cpp

```diff
@@ -810,13 +820,26 @@ std::vector<DebugRow> createDebugDataOfModState(const EvaluatorState &es)
         }
     };
 
-    rec(0, false);
-    lua_pop(es.L, -1);
+    std::vector<std::string> tablesList = {es.stateName, "shared"};
+    for (const auto &t : tablesList)
+    {
+        lua_getglobal(es.L, t.c_str());
+        if (!lua_istable(es.L, -1))
+        {
+            lua_pop(es.L, -1);
+            rows.emplace_back(0, "Error", "Not a Table");
+            return rows;
```

Comment:
Now this is a loop we may not want to do a return here since there's more recursion to be done. Probably we want to put the rec into an else so we can at least do the other table.

but honestly if one isn't there the other probably isnt' either so feel free to ignore this

Reply:
Maybe a `continue;` would be nicer then instead of that nested return?

Reply:
Yes that sounds correct to me.